README.md

```diff
@@ -1,70 +1,49 @@
-# Getting Started with Create React App
+# Anime-facts-webapp
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+A mobile web application that checks a list of Animes that is been created from the data fetched from a Anime facts API.
 
-## Available Scripts
+- ![Elements](https://user-images.githubusercontent.com/29541335/142738592-a00d625c-9e33-4e81-a720-dd5f72579f7b.png)
 
-In the project directory, you can run:
-
-### `yarn start`
-
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
-
-The page will reload if you make edits.\
-You will also see any lint errors in the console.
-
-### `yarn test`
-
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
-
-### `yarn build`
-
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+- ![Details](https://user-images.githubusercontent.com/29541335/142738577-b8072432-6415-40d1-b8e5-3cd1c21925b9.png)
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+## Built With
 
-### `yarn eject`
+- HTML
+- CSS
+- JAVASCRIPT
+- React && Redux
+## Live Demo
+[Live Demo Link](https://dreamy-booth-15f786.netlify.app/)
```

Comment:
- [ ] The provided Live demo link in your ReadMe is displaying a project other than what this project displays. Kindly ensure that your project is correctly deployed and the appropriate link is included in your ReadMe.

**Current Live Demo link (https://dreamy-booth-15f786.netlify.app/):**
![image](https://user-images.githubusercontent.com/12745474/142739401-612eb55b-e037-4ec9-b8aa-7fa292e2ffce.png)

**Current repository project:**
![image](https://user-images.githubusercontent.com/12745474/142739442-7cf31038-4cbf-4fb9-8e81-3206bb6e3f3d.png)

Reply:
I will do that, and sorry as I copied the template and forgot to change those.